apps/explorer/src/components/InlineErrorMessage.tsx

```diff
@@ -26,7 +26,9 @@ const InlineErrorMessage = ({ message, code }: { message?: string; code?: number
   if (!message) {
     shownMessage = "Something's wrong"
   } else {
-    shownMessage = message
+    // Clean the error message by removing details that could be used to trick users
+    const regex = /\(([^()]|\([^()]*\))*\)/g
```

Comment:
Can you please give me an explanation of what this regex does?

Reply:
Also, shouldn't we do the clean up in `getHumanReadableError` so that none of our apps get affected?

Reply:
Also, we should use `getHumanReadableError` in the `TransactionInfoPage`, like we do in the `BlockInfoPage` before displaying the message

Reply:
Let's have a quick call to go through this!